### ISSUE TITLE
Module request: SortMeRNA

### ISSUE THREAD

**Issue by lfaller** [ewels/MultiQC#421]
Hi Phil,

Thanks for providing the MultiQC tool -- it really is making my life a whole lot easier!

I would like to suggest adding a SortMeRNA tool. We run it routinely to identify and remove ribosomal RNAs after sequencing RNAseq data. 

Tool webpage: http://bioinfo.lifl.fr/RNA/sortmerna/
Example of a log file: https://www.dropbox.com/s/udwi7bnmbuuwieo/sortmerna.log?dl=0

The important part is the line: `Total reads failing E-value threshold = 4394708 (98.90%)`

This tells us that 98.9% of the reads analyzed were NOT rRNA, so those can be included in downstream analyses (ie a DE expression analysis). rRNA reads in this case can be thought of as noise, because they don't add anything interesting to the analysis.

It would be good to have an overview of these values for all the samples in one plot, so we can easily identify samples that might not yield good results down the line because they had, for example, large percentages of rRNA.

Unfortunately, the name of the log file by default depends on another input argument (`--aligned`), but it would always end in `.log`:

To get the sample log file I linked to above, I ran this command:

```
/home/lina/sortmerna/sortmerna --ref /home/lina/sortmerna/rRNA_databases/silva-bac-16s-id90.fasta,/home/lina/sortmerna/index/silva-bac-16s-db:/home/lina/sortmerna/rRNA_databases/silva-bac-23s-id98.fasta,/home/lina/sortmerna/index/silva-bac-23s-db:/home/lina/sortmerna/rRNA_databases/silva-arc-16s-id95.fasta,/home/lina/sortmerna/index/silva-arc-16s-db:/home/lina/sortmerna/rRNA_databases/silva-arc-23s-id98.fasta,/home/lina/sortmerna/index/silva-arc-23s-db:/home/lina/sortmerna/rRNA_databases/silva-euk-18s-id95.fasta,/home/lina/sortmerna/index/silva-euk-18s-db:/home/lina/sortmerna/rRNA_databases/silva-euk-28s-id98.fasta,/home/lina/sortmerna/index/silva-euk-28s:/home/lina/sortmerna/rRNA_databases/rfam-5s-database-id98.fasta,/home/lina/sortmerna/index/rfam-5s-db:/home/lina/sortmerna/rRNA_databases/rfam-5.8s-database-id98.fasta,/home/lina/sortmerna/index/rfam-5.8s-db --reads 25839_merged.fastq --num_alignments 1 --fastx --aligned Sample_2055324_ribo/sortmerna_out/2055324_ribo_rrna_reads --other Sample_2055324_ribo/sortmerna_out/2055324_ribo_rrna_free_reads.fastq fastq tq --log -v --blast 1 
```

In this case, the log file is named `Sample_2055324_ribo/sortmerna_out/2055324_ribo_rrna_reads.log`.

In any case, this would be a great additional module -- thanks!

**Comment by ewels** [comment]
Hi @lfaller,

Thanks for the detailed issue and example data! I've not heard of this tool before, looks interesting..

With other modules I've added a column to the _General Statistics_ table for `% rRNA`. So for this it would be `100 - reads_failing`. Would that do the job?

Phil

**Comment by ewels** [comment]
Sorry, just figured that this tool isn't going to be rRNA specific, so my suggestion won't be suitable for everyone I guess. I'll probably stick with a barplot graph as you suggest instead then.

Phil

**Comment by lfaller** [comment]
Technically, the tool is not rRNA specific (depends on which databases you provide) but most people do use it to filter out rRNA.

We could either show "% of rRNA filtered out" or "% non-rRNA identified"

Thanks for considering to add this as a module!

**Comment by ewels** [comment]
Closed in #448 by @bschiffthaler
